shared/common-adapters/plaintext-usernames.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React, {Component} from 'react'
+import Text from './text'
+import shallowEqual from 'shallowequal'
+import {isMobile} from '../constants/platform'
+
+import type {Props} from './plaintext-usernames'
+
+const inlineStyle = isMobile
```

Comment:
There's some duplication here with usernames.js...should I try and factor those out?